Changelog.md

```diff
@@ -11,6 +11,7 @@
 ## Features
 * Add client-side cropping of profile image uploads [#7581](https://github.com/diaspora/diaspora/pull/7581)
 * Add client-site rescaling of post images if they exceed the maximum possible size [#7734](https://github.com/diaspora/diaspora/pull/7734)
+* Add a manifest.json file as a first step to make diaspora* a Progressive Web App [#7997](https://github.com/diaspora/diaspora/issues/7997)
```

Comment:
Please don't add changelogs entries (this can be merged to the `next-minor`, so it's even in the wrong section ;) ).